pkg/grpc/labelstore/client/client.go

```diff
@@ -29,11 +29,22 @@ func NewClient(conn *grpc.ClientConn, logger logging.Logger) Client {
 	}
 }
 
+// this interface is just to make the compiler assert that our functions match
+// those in the direct consul applicator
+type client interface {
+	WatchMatches(selector klabels.Selector, labelType labels.Type, _ time.Duration, quitCh <-chan struct{}) (chan []labels.Labeled, error)
```

Comment:
so I take it these two WatchMatches *in this file* must now stay in sync?

Easier than checking checking whether it's in sync with a *different file* though, so I guess